.github/workflows/main.yml

```diff
@@ -0,0 +1,69 @@
+name: Serverless Plugin For Each CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  Setup-Env:
+     runs-on: ubuntu-latest
```

Comment:
```suggestion
     runs-on: [self-hosted, AWS, Linux]
```

Reply:
this is a public repo, no need to use self-hosted runners

Reply:
Right 👍